src/write_outputs/write_vre_stor.jl

```diff
@@ -434,7 +462,25 @@ function write_vre_stor_discharge(path::AbstractString,
         end
     end
 
-    # Solar generation of co-located resource dataframe
+    # Electrolyzer consumption of co-located resource dataframe
+    if !isempty(ELEC)
+        dfVP_VRE_STOR = DataFrame(Resource = inputs["RESOURCE_NAMES_ELEC"],
+            Zone = inputs["ZONES_ELEC"],
+            AnnualSum = Array{Union{Missing, Float32}}(undef, size(ELEC)[1]))
+        elec_vre_stor = value.(EP[:vP_ELEC]).data
+        if setup["ParameterScale"] == 1
+            elec_vre_stor *= ModelScalingFactor
+        end
+        dfVP_VRE_STOR.AnnualSum .= elec_vre_stor * inputs["omega"]
+
+        filepath = joinpath(path, "vre_stor_elec_power_consumption.csv")
```

Comment:
The filenames for wind and solar in this function are of the type "vre_stor_<resource>_power.csv". Can we also make the electrolyzer part follow the same pattern?
```suggestion
        filepath = joinpath(path, "vre_stor_elec_power.csv")
```